types/reply.d.ts

```diff
@@ -28,16 +28,17 @@ export type ResolveReplyTypeWithRouteGeneric<RouteGenericReply, Code extends Rep
  * FastifyReply is an instance of the standard http or http2 reply types.
  * It defaults to http.ServerResponse, and it also extends the relative reply object.
  */
-export interface FastifyReply<
+export type FastifyReply<
```

Comment:
As above, we should avoid converting an `interface` to a `type` (especially considering it's used everywhere).